packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -43,7 +44,21 @@ export function addTooltipOptions<T extends OptionsWithTooltip>(
       settings: {
         options: sortOptions,
       },
-    })
+    });
+
+  if (setProximity) {
+    builder.addNumberInput({
```

Comment:
should we default this to `15` as per https://github.com/grafana/grafana/pull/81421/files#diff-8d24e749f24ed73b06db210b373dae9a91e859ea27635d0842e4fb308d288529R562?